MAINTAINERS.yml

```diff
@@ -2288,6 +2291,22 @@ Intel Platforms (Xtensa):
   labels:
     - "platform: Intel ADSP"
 
+Intel Platforms (Agilex):
+  status: maintained
+  maintainers:
+    - gdengi
+  collaborators:
+    - nbalabak
+    - chongteikheng
```

Comment:
@gdengi Hi. The collaborator added here, `chongteikheng`, does not seem to be a valid GitHub account. Please submit a patch to either update or remove the username from the MAINTAINERS file.

Reply:
Hello @stephanosio, sure will submit a patch to update the account name here. Thank you.

Reply:
Hello @stephanosio, FYI, created a PR for the same. https://github.com/zephyrproject-rtos/zephyr/pull/65190